### PR TITLE
Update to Security for SSL importing.

### PR DESCRIPTION
**Pull request by Tigatok**
Thank you so much for contributing code to lando!

We will get to your PR as soon as we can but in the meantime you might want to
check to make sure you have done the following. **The more boxes you can check
the easier it will be for us to merge in your changes with confidence**

- [x] My code passes relevant CI status checks
- [x] My code includes a test if applicable ([see here](https://docs.devwithlando.io/dev/testing.html))
- [x] My code includes an update to the `CHANGELOG` ([see here](https://github.com/kalabox/lando/tree/master/docs/changelog))
- [x] My code includes documentation updates if relevant.
- [x] I have pinged [a project committer](https://docs.devwithlando.io/contrib/contributing.html#committers) when I think my code is ready for prime time.

If you are unclear about any of the above please add comments below so we can
improve our process.

### CODE REVIEW

**Comment by ElitesRwe** [comment]
Slow response time

File: docs/config/security.md [hunk 1]
```diff
@@ -55,3 +55,18 @@ sudo update-ca-certificates
...
+Currently works for: Version 67.0.3396.79 (Official Build) (64-bit)
+
+1. Navigate to Settings (the 3 dots in the upper right hand corner).
+2. Use the search bar to search for `SSL`.
+3. Click `Manage Certificates`.
+4. Click on `Authorities`.
+5. Click `Import`.
+6. Navigate to your `~/.lando/certs/lando.pem` key, and import it.
+7. Check all settings that apply.
+8. Flush browser cache and naviate to your Lando website.
```
Comment (serundeputy) [inline docs/config/security.md]: ~~naviate~~ should be navigate?

Comment (Tigatok) [reply docs/config/security.md]: Fixed, thanks!

**Comment by Tigatok** [comment]
I believe Mac's handle the certs through Keychain. I think importing a self-signed cert is necessary on ubuntu.

**Comment by pirog** [comment]
@Tigatok exactly. what im implying here is that it makes more sense to me for importing into the browser be nested under "debian" instead of its own thing.

**Comment by Tigatok** [comment]
Gotcha, thanks @pirog I'll see if I can update that to be more OS specific. I think I can just leave MACOS as 'should be imported automatically', eh?

**Comment by pirog** [comment]
@Tigatok 👍

**Comment by dustinleblanc** [comment]
I'm going to go ahead and close this as it's been pending tweaks for over a month. @Tigatok if you want to make those tweaks and re-open, that'd be great, thanks!

**Pull request closed without merging**